blurp-app/src/components/searchbar.jsx

```diff
@@ -3,14 +3,20 @@ function SearchBar() {
     return (
         <>
             <form class="absolute w-full flex items-center justify-center py-2">   
-            <div class="relative w-2/5">
-                <div class="searchbar-icon">
-                    <svg aria-hidden="true" class="w-5 h-5 text-gray-500 dark:text-gray-500" fill="currentColor" viewBox="0 0 20 20"><path fill-rule="evenodd" d="M8 4a4 4 0 100 8 4 4 0 000-8zM2 8a6 6 0 1110.89 3.476l4.817 4.817a1 1 0 01-1.414 1.414l-4.816-4.816A6 6 0 012 8z" clip-rule="evenodd"></path></svg>
+                <div class="relative w-2/5">

```

Comment:
small nitpick but I will make a pull request soon about this to other files but going forward please change `class=` to `className=`. In JSX files using `class=` results in an error message in the console which can be annoying when reading the console. 

More info: https://stackoverflow.com/questions/30968113/warning-unknown-dom-property-class-did-you-mean-classname

Reply:
Oh, thanks for telling me about that, I didn't realize that's what those annoying warnings were. I fixed that here and will fix the issue in my other files as well